pkg/kv/kvserver/store_raft.go

```diff
@@ -61,9 +65,20 @@ func (q *raftReceiveQueue) drainLocked() ([]raftRequestInfo, bool) {
 	}
 	infos := q.mu.infos
 	q.mu.infos = nil
+	q.acc.Clear(context.Background())
 	return infos, true
 }
 
+func (q *raftReceiveQueue) Delete() {
+	q.mu.Lock()
+	defer q.mu.Unlock()
+	q.drainLocked()
+	if err := q.acc.ResizeTo(context.Background(), 0); err != nil {
+		panic(err) // TODO(tbg)
```

Comment:
Do we need to fix this?

Reply:
No, ResizeTo never returns an error when passed zero. I'll fix the comment.